### ISSUE TITLE
[RFC] Drop Travis Support

### ISSUE THREAD

**Issue by vrom911** [kowainik/summoner#538]
It seems to be rarely used nowadays, so we can get rid of the pressure of additional maintenance and testing.

**Comment by chshersh** [comment]
@vrom911 I completely agree with you 👍🏻 Maybe not in the next release (to give people some time to migrate), but I believe we can remove Travis in the next one after the upcoming 🧹
